lib/registrant.rb

```diff
@@ -0,0 +1,18 @@
+class Registrant
+  attr_accessor :license_data
+  attr_reader :name, :age, :permit
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {:written=>false, :license=>false, :renewed=>false}
+  end
+
+  def permit?
+    @permit
+  end
+
+  def earn_permit
+    @permit = true
```

Comment:
This is great. We could also add in a guard clause like
```ruby
@permit = true if @age >= 15
```
Which would really ensure that a registrant could not get a permit unless eligible.